addons/auth/package.json

```diff
@@ -21,52 +21,52 @@
     "doc:toc": "doctoc README.md"
   },
   "dependencies": {
-    "@embroider/test-setup": "^0.48.1",
-    "ember-auto-import": "^2.4.2",
-    "ember-cli-babel": "^7.26.10",
-    "ember-cli-htmlbars": "^6.1.0",
+    "ember-cli-babel": "^7.26.11",
+    "ember-cli-htmlbars": "^6.1.1",
     "ember-fetch": "^8.1.1",
     "ember-simple-auth": "^4.2.2"
   },
   "devDependencies": {
-    "@babel/eslint-parser": "^7.18.2",
-    "@babel/plugin-proposal-decorators": "^7.18.6",
+    "@babel/eslint-parser": "^7.19.1",
+    "@babel/plugin-proposal-decorators": "^7.20.2",
     "@ember/optional-features": "^2.0.0",
     "@ember/test-helpers": "^2.8.1",
+    "@embroider/test-setup": "^1.8.3",
     "@glimmer/component": "^1.1.2",
     "@glimmer/tracking": "^1.1.2",
     "broccoli-asset-rev": "^3.0.0",
     "doctoc": "^2.2.0",
-    "ember-cli": "^4.4.0",
+    "ember-auto-import": "^2.4.3",
+    "ember-cli": "^4.4.1",
     "ember-cli-dependency-checker": "^3.3.1",
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
     "ember-cookies": "^0.5.2",
     "ember-disable-prototype-extensions": "^1.1.3",
-    "ember-export-application-global": "^2.0.1",
     "ember-load-initializers": "^2.1.2",
-    "ember-page-title": "^7.0.0",
     "ember-qunit": "^6.0.0",
     "ember-resolver": "^8.0.3",
-    "ember-source": "~4.0.1",
+    "ember-source": "~4.4.4",
     "ember-source-channel-url": "^3.0.0",
-    "ember-template-lint": "^4.14.0",
+    "ember-template-lint": "^4.16.1",
+    "ember-try": "^2.0.0",
     "eslint": "^8.18.0",
     "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-ember": "^10.6.1",
+    "eslint-plugin-ember": "^11.1.0",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-qunit": "^7.3.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-qunit": "^7.3.1",
     "loader.js": "^4.7.0",
     "npm-run-all": "^4.1.5",
     "pretender": "^3.4.3",
-    "prettier": "^2.5.1",
-    "qunit": "^2.17.2",
+    "prettier": "^2.7.1",
+    "qunit": "^2.19.2",
     "qunit-dom": "^2.0.0",
-    "webpack": "^5.65.0"
+    "webpack": "^5.74.0"
   },
   "engines": {
-    "node": "12.* || 14.* || 16.*"
+    "node": "14.* || 16.* || >= 18"
```

Comment:
I was thinking on NOT commiting within Ember 4 work any changes to node support.

I do believe, it is better to manage node version updates/deprecations in its unique PR.

Why?
- It also requires changes on documentation (such as Readme).
- It requires testing the CI and Vercel deployments are ok with this changes (in the past we run in some issues).
- Facilitates testing and validation focusing on just one thing (node in this case).

Reply:
Yeah that's fine.  It looks like node 12 is supported through ember-cli 4.6:  https://github.com/ember-cli/ember-cli/blob/master/docs/node-support.md

Reply:
Yep, I will create a separate jira task to take care of node depreactions, node 12 is not supported anymore and 18 is the current LTS, [more infor here](https://github.com/nodejs/release#release-schedule)